cenpy/legacy_tiger.py

```diff
@@ -0,0 +1,229 @@
+import geopandas as gpd
+from warnings import warn
+from gdal import osr, ogr
+import shutil
+
+from urllib.request import urlopen
+from zipfile import ZipFile
+import os
+import tempfile
+
+
+class FTPConnection(object):
+
+    def __init__(self, year, statenum, stateabbr, county):
```

Comment:
As you identify, I'll try to make edits so that this refers to only the `year`, and `query` refers to states & counties. 

We can look up `stateabbr` in `cenpy.explorer.fips_table`, so it shouldn't be necessary explicitly. 

Also, still working on crawling the 1990 & 2000 variable endpoints for the table of FIPS codes nesting from state->county->tract->block using `cenpy.tools.national_to_block`. I set up the crawl, but sometimes it fails mid-way through because of rate limits or disconnections. I haven't had the time to sit down and set the backoff right yet.

Reply:
last week i was having some trouble finding the resources via the http protocol, but returning to it this week, it doesn't seem to be a problem. And in fact, the connection is a lot better

Reply:
there also may be a memory leak or something in tiger -> gdf conversion because it will frequently crash my kernel. still investigating